librad/tests/smoke/saturation.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+use librad::identities::payload;
+use librad_test::{
+    logging,
+    rad::{identities::TestProject, testnet},
+};
+
+#[tokio::test]
+async fn saturate_a_peer_with_projects() {
+    logging::init();
+
+    const NUM_PEERS: usize = 2;
+    const NUM_PROJECTS: usize = 11;
```

Comment:
This needs to be 12+.

Reply:
Is 12 a magic number there, yes?

Reply:
There's the initial project too :) So it adds up to 12. Can bump it though if you think that'd make more sense?

Reply:
> Is 12 a magic number there, yes?

Yup, the seed won't have any fetching logs nor git tracing when the 12th project was announced

Reply:
My bets are on fd exhaustion...

Reply:
I believe @xla took a look at the fd's and didn't see anything out of the ordinary, but I'll let him chime in